src/clj/ataru/kk_application_payment/kk_application_payment_status_updater_job.clj

```diff
@@ -123,6 +124,15 @@
                                              "kk-application-payment-person-status-update-job"
                                              {:person_oid person-oid :term term :year year}))))
 
+(defn start-update-kk-payment-status-for-application-key-job
```

Comment:
Tätähän kutsutaan nyt vaan application keyllä, onko application-id -tuki ylipäänsä tarpeellinen?

Reply:
Hakemuksen id:llä tuota kutsutaan person-integrationin start-jobs-for-application:ista luomalla suoraan halutunlainen jobi. Totesin, että henkilön päivittämisen yhteydessä ei kannata tehdä ylimääräistä application-kantakyselyä, joten parempi hoitaa id -> key vasta tuon päivitysjobin yhteydessä.

Nyt kun asiasta mainitsit, se kannattanee tehdä näkyväksi vaikka start-update-kk-payment-status-for-application-id-job -tyyppisellä funktiolla jota kutsutaan jobin suoraan luomisen sijaan.

Reply:
Korjailin tämän suoraan OK-hakemusmaksut-branchiin.